storage/rook/minio/object-store.yaml

```diff
@@ -6,19 +6,19 @@ metadata:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: minio-my-store-access-keys
+  name: minio-object-store-access-keys
   namespace: rook-minio
 type: Opaque
 data:
-  # Base64 encoded string: "TEMP_DEMO_ACCESS_KEY"
-  username: VEVNUF9ERU1PX0FDQ0VTU19LRVk=
-  # Base64 encoded string: "TEMP_DEMO_SECRET_KEY"
-  password: VEVNUF9ERU1PX1NFQ1JFVF9LRVk=
+  # Base64 encoded string: "minio"
+  username: bWluaW8=
+  # Base64 encoded string: "minio123"
+  password: bWluaW8xMjM=
```

Comment:
Is there any special meaning for the default username/password?

Reply:
@evans-ye No. Jay used the user/password from his demo: https://github.com/jayunit100/apachecon-2019-bigtop3x/blob/master/presto3-minio/coordinator/minio.properties